bookkeeper-server/src/main/java/org/apache/bookkeeper/replication/Auditor.java

```diff
@@ -184,6 +186,7 @@ private void submitShutdownTask() {
             executor.submit(new Runnable() {
                     public void run() {
                         synchronized (Auditor.this) {
+                            LOG.info("Shutting down Auditor's Executor");
```

Comment:
Can't we just say - Shutting down the Auditor? 'executor' adds any value?

Reply:
there is separate Auditor.shutDown method, i don't want to mix them up.

Anyhow I feel some more logging is needed for this method Auditor.submitShutdownTask